### PR TITLE
Get rid of deprecation warning about :text format in render

### PR DESCRIPTION
**Pull request by hallelujah**
Remove deprecation of render text: 'Something'
Uses render plain: 'Something' instead

### CODE REVIEW

File: app/controllers/provider/admin/webhooks_controller.rb [hunk 1]
```diff
@@ -24,7 +24,7 @@ def show
         if @webhook
           @ping_response = @webhook.ping
         else
-          render :status => :forbidden, :text => 'Nowhere to ping'
+          render :status => :forbidden, :plain => 'Nowhere to ping'
```
Comment (Martouta) [inline app/controllers/provider/admin/webhooks_controller.rb]: :eyes: 
```suggestion
          render status: :forbidden, plain: 'Nowhere to ping'
```

File: lib/developer_portal/app/controllers/developer_portal/cms/new_content_controller.rb [hunk 2]
```diff
@@ -54,7 +54,7 @@ def show
 
       # CMS::Page#mime_type is parsed and valid Mime::Type
       # in case it can't be parsed (nil, something invalid) returns 'text/html'
-      render :layout => false, :content_type => page.mime_type, :text => renderer.content
+      render :layout => false, :content_type => page.mime_type, :plain => renderer.content
```
Comment (Martouta) [inline lib/developer_portal/app/controllers/developer_portal/cms/new_content_controller.rb]: :eyes: 
```suggestion
      render layout: false, content_type: page.mime_type, plain: renderer.content
```

File: lib/developer_portal/app/controllers/developer_portal/cms/new_content_controller.rb [hunk 3]
```diff
@@ -54,7 +54,7 @@ def show
 
       # CMS::Page#mime_type is parsed and valid Mime::Type
       # in case it can't be parsed (nil, something invalid) returns 'text/html'
```
Comment (Martouta) [inline lib/developer_portal/app/controllers/developer_portal/cms/new_content_controller.rb]: This comment does not match [what it does](https://guides.rubyonrails.org/upgrading_ruby_on_rails.html#rendering-content-from-string) anymore.
`render :plain` returns `text/plain` and not `text/html` and `render :text` does. For that response type, the new way would be using `render :body`.

Comment (hallelujah) [reply lib/developer_portal/app/controllers/developer_portal/cms/new_content_controller.rb]: But actually it should not be text/html as it is only text ...

Comment (hallelujah) [reply lib/developer_portal/app/controllers/developer_portal/cms/new_content_controller.rb]: Oh you are right, render :body does not set the mime :)

I will change thanks!

**Pull request merged**